src/components/header.component.vue

```diff
@@ -0,0 +1,395 @@
+                <img src="../assets/Time_icon.svg" />
+                <div class="SearchHistory__GeneratedAtText">13:15 기준</div>
+              </div>
+            </h3>
+          </div>
+          <div class="PopularSearchKeywordList__Row">
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(-5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number" v-if="index < 5">{{
+                  index + 1
+                }}</span>
+                <span class="CandidateItem__Text" v-if="index < 5">{{
+                  Item
+                }}</span>
+              </div>
+            </div>
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number">{{ index + 6 }}</span>
+                <span class="CandidateItem__Text">{{ Item }}</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Header',
+  data: () => {
+    return {
+      isShowCategory: false,
+      isShowCartText: true,
+      DummyData: {
+        recommended: [
+          '오늘의 특가',
+          '아이패드0원',
+          'TOP20',
+          '1+1클래스',
+          '여름 취미 추천',
+          '감사제 특가',
+          '10만원할인',
+        ],
+        popular: [
+          '아이패드',
+          '일러스트',
+          '이모티콘',
+          '드로잉',
+          '포토샵',
+          '뜨개질',
+          '레진',
+          '주식',
+          '포토크리에이트',
+          '가전주부 PICK',
+        ],
+      },
+    }
+  },
+  computed: {
+    SearchInputHandler: function() {
+      if (this.isShowCategory == true) {
+        return {
+          width: '676px',
+          margin: 'auto',
+          'align-items': 'center',
+        }
+      } else {
+        return {
+          width: '420px',
+        }
+      }
+    },
+    SearchContainerHandler: function() {
+      if (this.isShowCategory == true) {
+        document.body.style.cssText = `overflow: hidden;`
+        return {
+          display: 'flex',
+          'flex-direction': 'row',
+        }
+      } else {
+        return {
+          position: 'realtive',
+          flex: '1 1 0%',
+        }
+      }
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+input::-ms-clear,
+input::-ms-reveal {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+input::-webkit-search-decoration,
+input::-webkit-search-cancel-button,
+input::-webkit-search-results-button,
+input::-webkit-search-results-decoration {
+  display: none;
+}
+
+.HeaderWrapper {
+  width: 100%;
+  height: 5.375rem;
+  .HeaderBox {
```

Comment:
새로운 선택자가 들어오면서 위 블록과 다른 스코프를 갖게 되었어요
따라서 한 줄 띄워주는 것이 가독성에 도움이 됩니다.
```suggestion

  .HeaderBox {
```